src/libs/ajax/OAuth2.ts

```diff
@@ -0,0 +1,20 @@
+import { fetchOrchestration } from 'src/libs/ajax/ajax-common';
+
+/*
+ This module is for all methods that talk to the oidc client or needed in order to talk to
+ the oidc client
+ */
+
+export type OidcConfig = {
```

Comment:
Nit: prefer `interface` over `type` for clearer error messages.

Not as applicable for return types, but it could still help in tests.

```suggestion
export interface OidcConfig {
```